sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCopilotPromoOptInStatus.md

```diff
@@ -0,0 +1,55 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Get-SPOCopilotPromoOptInStatus
+applicable: SharePoint Online
+title: Get-SPOCopilotPromoOptInStatus.md
+schema: 
+author: swathi.iruvanti
+ms.author: siruvanti
+ms.reviewer:
+---
+# Get-SPOCopilotPromoOptInStatus
+
+## SYNOPSIS
+
+Returns the true or false Opt-In Copilot promo status 
```

Comment:
could you come up with a better string here?

also sentence should end with `.`